strax/context.py

```diff
@@ -1361,6 +1358,106 @@ def _apply_function(self, data, targets):
             data = function(data, targets)
         return data
 
+    def _copy_to_frontend(self, run_id, target,
+                          target_frontend_id=None, rechunk=False):
+        """
+        Copy data from one frontend to another
+        :param run_id: run_id
+        :param target: target datakind
+        :param target_frontend_id: index of the frontend that the data should go to
+        in context.storage. If no index is specified, try all.
+        :param rechunk: allow re-chunking for saving
+        """
+        if not self.is_stored(run_id, target):
+            raise strax.DataNotAvailable(f'Cannot copy {run_id} {target} since it '
+                                         f'does not exist')
+        if len(strax.to_str_tuple(target)) > 1:
+            raise ValueError('_copy_to_frontend only works for ')
+        if target_frontend_id is None:
+            target_sf = self.storage
+        elif len(self.storage) > target_frontend_id:
+            # only write to selected other frontend
+            target_sf = [self.storage[target_frontend_id]]
```

Comment:
I am wondering if it would not be a bit neater to add a `fornt_end_id` to the properties of every [forntned](https://github.com/AxFoundation/strax/blob/3dece573fdfd44955cc69d95dbdd790451f3d933/strax/storage/common.py#L120). In that case one could precisely distinguish between frontends. However, this would also require some additional checks, e.g. that the same id is not specified twice.

Reply:
What would be the gain? Now it's just the index in st.storage which is a list. Using a number is quite simple. 

Could try to match the __repr__ or the input but that sounds more convoluted. Could do though because you than don't by accident fill the wrong sf.

Reply:
I agree, I think it is not really needed and I am fine with the PR as is. 

> Could do though because you than don't by accident fill the wrong sf.

This was also my concern, but I am not sure if the "risk" justifies the extra work.